ONSdigital/eq-survey-runner
# Update sinon, mocha and livereload dependencies

### What is the context of this PR?
Fixes #1132.

Updates versions of mocha and sinon to newer versions.

Updates gulp-livereload to gulp-refresh. Refresh is a fork of livereload which is actively maintained. Livereload is now considered unmaintained.

Adds a new feature flag to enable livereload. When this flag is enabled the CSP policy is altered to allow livereloading and the browsersync script is added to the base template.

### How to review 
All functional tests should pass

To test live reloading:
- Run the app locally
- Also run `yarn start`
- Start a questionnaire
- A message should say "Browsersync: connected"
- Make a change to a JS, SCSS or HTML file
- The browser should refresh and reflect the change
